geolookup/geolookup.go

```diff
@@ -2,261 +2,83 @@ package geolookup
 
 import (
 	"context"
-	"encoding/json"
-	"fmt"
-	"math"
-	"net/http"
-	"os"
 	"sync"
 	"time"
 
 	"github.com/getlantern/eventual/v2"
-	geo "github.com/getlantern/geolookup"
 	"github.com/getlantern/golog"
 
-	"github.com/getlantern/flashlight/v7/ops"
-	"github.com/getlantern/flashlight/v7/proxied"
+	proxyconfig "github.com/getlantern/flashlight/v7/config/proxy"
 )
 
 var (
 	log = golog.LoggerFor("flashlight.geolookup")
 
-	refreshRequest = make(chan interface{}, 1)
-	currentGeoInfo = eventual.NewValue()
-	watchers       []chan bool
-	persistToFile  string
-	mx             sync.Mutex
-	roundTripper   http.RoundTripper
-)
+	// _country and _ip are eventual values that hold the current country and IP as strings.
+	_country = eventual.NewValue()
+	_ip      = eventual.NewValue()
```

Comment:
There's a reason for the `_` on the name? I guess we could call them country and ip directly or add a eventual prefix

Reply:
There used to be. I have since changed some things and I forgot they were named that way. I'm going to change them.